src/auth/providers.js

```diff
@@ -7,6 +7,8 @@ import { uriEncodeObject } from '../util';
  * @namespace
  */
 function anonProvider(auth) {
+  const providerType = 'anon';
```

Comment:
[nit] We should probably make this exactly the same name/type that we use in the route for every provider. i.e. `anon` -> `anon-user`.

Reply:
I take this back. We should really haver a constant here that is used in createProviders as well.